docs/source/_json/collection.json

```diff
@@ -5,8 +5,19 @@ HTTP 200 OK
 content-type: application/json
 
 {
+  "@components": {
```

Comment:
@tisto regarding nesting I'm fine with both options.

Expansion works on "elements". Thus you can expand breadcrumbs and navigation separately. "Components" is not an expandable element.